### ISSUE TITLE
Strategy for downsampling actual data for training

### ISSUE THREAD

**Issue by MoHeydarian** [galaxyproject/training-material#1482]
Can you share your very nice strategy and workflow for downsampling real sequencing data that you described to us in person @lldelisle?

**Comment by lldelisle** [comment]
To generate the small input dataset this is what I did:
1) Take only 20M R1 and R2 reads from the original dataset using seqtk sample (which is about 200M R1 and R2).
2) Evaluate that chr22 is about 1% of the other numbered chr
3) run the workflow: https://raw.githubusercontent.com/lldelisle/myWorkflows/master/Galaxy-Workflow-MakeAFakeInput.ga which:
a) Take randomly the id of 1% of the input R1 fastq
b) Take all id of reads which map to chr22
c) Concatenate the two and remove duplicates
d) use seqtk subseq to select R1 and R2...
We are now down to 380k reads.

**Comment by bebatut** [comment]
Nice! Thanks @lldelisle for sharing. Could you share the process on the [tutorial for tutorial creation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html#a-toy-dataset) where we tried to explain the concept of toy dataset? The sources are [there](https://github.com/galaxyproject/training-material/edit/master/topics/contributing/tutorials/create-new-tutorial/tutorial.md).
It would be awesome! 😄

**Comment by lldelisle** [comment]
I try to do it when I am done with the ATAC-seq tutorial as I cannot fork 2 repositories with the same name...

**Comment by nsoranzo** [comment]
@lldelisle You don't need 2 repositories, in Git you use "branches" to work on multiple lines of development at the same time: https://git-scm.com/book/it/v2/Git-Branching-Basic-Branching-and-Merging

**Comment by lldelisle** [comment]
@nsoranzo, many thanks, I was using branches but I was not using 2 remotes. This solved my issue.

@bebatut, 
Do you know where the current example of toy dataset (16S) comes from?
Many thanks,

**Comment by lldelisle** [comment]
I wrote a [PR](https://github.com/galaxyproject/training-material/pull/1513).

**Comment by bebatut** [comment]
@lldelisle Thanks for the PR.

The 16S toy dataset was from the tests of a Galaxy tool.

For another case (I think it was the the [metagenomics general tutorial](https://galaxyproject.github.io/training-material/topics/metagenomics/tutorials/general-tutorial/tutorial.html)), I used a similar approach to yours.
I took a real dataset from a know database and I ran the full workflow on it. Then for each step (starting from the last steps and going to the first one), I extracted randomly out of the outputs some sequences id for good and bad sequences, trying to keep similar percentage of bad and good sequences). Then I extracted the raw sequences corresponding to the ids to build my toy dataset.
